recipes/mongomock/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{%set name = "mongomock" %}
+{%set version = "3.6.0" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/mongomock/{{ name }}.git
+  git_rev: {{ version }}
```

Comment:
Can we please change this to a tarball and have a checksum?

e.g. https://github.com/mongomock/mongomock/archive/{{ version }}.tar.gz